docs/operator_parameters.md

```diff
@@ -0,0 +1,175 @@
+Postgres Operator Configuration
+===============================
+
+Postgres operator is configured via a ConfigMap defined by the
+*CONFIG_MAP_NAME* environment variable. Variable names are delimiter-separated
+words.
+
+Available Configuration Parameters
+----------------------------------
+
+* **etcd_host**
+  Etcd host:port for Patroni. Not required when Patroni native Kubernetes
+  support is used.
+
+* **docker_image**
+  Spilo docker image for Postgres instances. For production, don't rely on a
+  default value, as it might be too old. Instead, build your own Spilo image
+  from the [github repository](github.com/zalando/spilo).
+
+* **workers**
+  number of working routines the operator spawns to process requests to
+  create/update/delete/sync clusters concurrently. Default is *4*.
+
+* **pod_terminate_grace_period**
+  Patroni pods will be [terminated
+  forcefully](https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods)
+  after this timeout. Default is '*5m*'
+
+### Users
+
+* **super_username**
+  postgres superuser name created by *initdb*. Default is '*postgres*'.
+
+* **replication_username**
+  postgres user name used for replication between instances. Default is
+  '*standby*'
+
+### Kubernetes resources
+* **pod_service_account_name**
+  service account used by Patroni running on individual Pods to communicate
+  with the operator. Required even if native Kubernetes support in Patroni is
+  not used, because Patroni will still try to keep pod labels in sync with the
+  instance role. A default value is *operator*.
+
+* **pod_service_account_definition**
+  operator will try to create the pod Service Account in the namespace that
+  lacks it using the YAML definition provided by this option. If not defined, a
+  simple definition which contains only the name will be used.
+
+* **watched_namespace**
+  operator watches for Postgres objects in the given namespace. If not
+  specified, the value is taken from the operator namespace. A special '*'
```

Comment:
> the value is taken from the operator namespace.

"the operator watches its own namespace" sounds better to me cause the original phrase may be understood as "the operator need to retrieve sth from his own namespace"